openapi/response.go

```diff
@@ -12,21 +14,31 @@ func (r *Response) GetName() string {
 	return r.parent.GetName() + r.renderer.sanitiseName(r.name) + "Response"
 }
 
-func (r *Response) getRef() string {
-	return ""
-}
+//func (r *Response) getRef() string {
+//	panic("Composed type should override this")
+//	return ""
+//}
 
 var _ Traversable = &Response{}
 
 func (r *Response) getChildren() map[string]Traversable {
-	children := map[string]Traversable{}
+	responses := map[string]Traversable{} // Where string is either `default` or an HTTP status code
 	for name, mediaType := range r.Content {
-		children[name] = &mediaType
+		if _, err := strconv.Atoi(name); err != nil || name == "default" {
+			responses[name] = &mediaType
+		} else {
+			panic("Response spec broken")
```

Comment:
If you intend for can users to see this then we should probably be more specific

Reply:
Not sure of the road forward here. Validation upfront would be first prize